part-spec/part_spec.md

```diff
@@ -773,6 +950,23 @@
     * optional
 
 ## test point
+* manufacturer
```

Comment:
I don't think we'd have these fields for a test point. We can also decide to just remove this from the part spec since it isn't really a part.